internal/view/assets/js/page/home.js

```diff
@@ -153,6 +156,12 @@ export default {
 		},
 	},
 	methods: {
+		clearHomePage() {
```

Comment:
I think this function should only clear the homepage, the bind to the event should be made in the `mounted` method to only run that once.

Reply:
I meant that the `this.$bus.on` should be on mount (if I understand Vue correctly) so that is binded only once when the component is loaded. Unsure if there's a better method for that.

The `clearHomePage` method should only `search=""` && `this.searchBookmarks()`. That should be cleaner.

Maybe we should make it work only if the user is already at the homepage and not coming from another page? That can be done later in another PR, though.

Reply:
This is what I was referring to, yes: 31b392ac3905ff2460c996939f427947bc994d45